src/bin_editor/Oni2_editor.re

```diff
@@ -120,14 +120,28 @@ let createWindow = (~forceScaleFactor, ~workingDirectory, app) => {
     | _ => true
     };
 
+  let icon =
+    switch (Revery.Environment.os) {
+    | Mac =>
+      switch (Sys.getenv_opt("ONI2_BUNDLED")) {
+      | Some(bundled) =>
+        switch (bundled) {
+        | "1" => None
+        | _ => Some("logo.png")
+        }
```

Comment:
I think we could simplify this - `Some(_) => None` (whenever the `ONI2_BUNDLED` env variable is set at all, just use `None`)

Reply:
Awesome I just changed that in the most recent commit. Thanks!